packages/core/stories/tokens/logo/Logo.stories.mdx

```diff
@@ -31,115 +31,98 @@ According the [public records on the Massachusetts state seal](https://www.sec.s
-
 <div className="ma__row">
-  <ButtonWithIcon text="SVG" icon={<IconDownload />} usage="secondary" size="small" capitalized href={`${STORYBOOK_CDN_PATH}/static/images/logo/stateseal-white.svg`} download />
-  <ButtonWithIcon text="PNG (200 x 200 px)" icon={<IconDownload />} usage="secondary" size="small" capitalized href={`${STORYBOOK_CDN_PATH}/static/images/logo/stateseal-white.png`} download />
+	<strong>White</strong>
+  <ColorSwatch value="#FFFFFF" width={20} height={20} copiable={false} inline />
 </div>
 
+The white variation of the Great Seeal should be used on a background that is a solid named color, a shade of any color, or a tint of 30% or less.
+
 <Canvas withSource="none">
   <Story story={stories.sealWhiteBGDark} />
   <Story story={stories.sealWhiteBGDark1} />
+	<ButtonWithIcon text="SVG" icon={<IconDownload />} usage="secondary" size="small" capitalized href={`${STORYBOOK_CDN_PATH}/static/images/logo/stateseal-white.svg`} download />
+  <ButtonWithIcon text="PNG (200 x 200 px)" icon={<IconDownload />} usage="secondary" size="small" capitalized href={`${STORYBOOK_CDN_PATH}/static/images/logo/stateseal-white.png`} download />
 </Canvas>
 
+<div className="ma__row">
+	<strong>Gray</strong>
+  <ColorSwatch value="#777777" width={20} height={20} copiable={false} inline />
+</div>
 
+The gray variation of the Great Seeal should be used only on a background that is a tint of 70% or greater of any named color.
 
+<Canvas withSource="none">
+  <Story story={stories.seal} />
+  <Story story={stories.sealBGLight} />
+  <Story story={stories.sealBGLight1} />
+	<ButtonWithIcon text="SVG" icon={<IconDownload />} usage="secondary" size="small" capitalizedhref={`${STORYBOOK_CDN_PATH}/static/images/logo/stateseal.svg`} download />
+  <ButtonWithIcon text="PNG (200 x 200 px)" icon={<IconDownload />} usage="secondary" size="small" capitalized href={`${STORYBOOK_CDN_PATH}/static/images/logo/stateseal.png`} download />
+</Canvas>
+
+<br />
 
 ## Accessibility & Best Practices
 
 ### Sizing
-When reproducing the state seal, it must remain proportionate and legible relative to its size. It should never be so small that it is no longer distinguishable, as it will no longer serve its main purpose of identifying official communications from the Commonwealth.
 
-| Medium  | Recommended Size |
-|---|---|
-|**Print**   | 1 in x 1 in  |
-|**Digital**  | 45px x 45px  |
+The state seal must remain proportionate (equal in width and height) and legible relative to its size. It should also not be very large since it's use is as an identifier and not a hero image.
 
-Exception to the rule: The Great Seal may appear smaller than 45px when a software application or website requires an icon of a specific size, such as a website “favicon.”
+| Medium | Recommended Size | Minimum size | Maximum size |
+|---|---|---|---|
+| **Print (in)** | 0.5 x 0.5 | 1 x 1 | 2 x 2 |
+| **Digital (px)** | 32 x 32 | 45 x 45 | 200 x 200 |
```

Comment:
the recommended sizes are smaller than the minimum sizes?

Reply:
oops. No that's wrong.

Reply:
Fixed